CONTRIBUTING.md

```diff
@@ -35,14 +35,14 @@ For Git commit messages, our principle is that `git log --pretty=oneline` should
 
 The following is an example of a well-formed commit, including the description (first line) and a body that includes changelog additions:
 
-      Introduced a new API for package management (#1311)
+      Fixes #1311
```

Comment:
That's not a useful first line. The previous one was better.

We tend to use the present tense in commit messages, so I'd suggest "Introduces" or "Introduce" over "Introduced".

```suggestion
      Introduces a new API for package management

      Fixes #1311.
```

Reply:
as above